homeassistant/components/switch/broadlink.py

```diff
@@ -68,6 +70,8 @@
     vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int
 })
 
+SAVE_PATH = os.path.expanduser('~')+'/.homeassistant/known_packets.yaml'
```

Comment:
This will not work. The config folder can be anywhere

Reply:
so, I try to save file more official,
such as
```
from homeassistant.config import load_yaml_config_file
```
or
```
from homeassistant.util.json import load_json, save_json
```